src/model/mod.rs

```diff
@@ -373,15 +381,15 @@ impl Factory {
                 y: 0.0,
                 uvlock: false,
                 weight: 1.0,
-
+                /*
```

Comment:
Also is this change intended?

Reply:
I have enabled clippy checks that emit warning for unused code. This change is intended because the code commented was not used, according to clippy report. Leaving dead code is a bad practice and later it will be hard to track - it was already kind of annoying because there were many places with such code. Therefore, it is best to comment such code and probably leave `TODO` to allow easier searching later.

At least that is the practice my employer uses.

Reply:
That's okay, but these TODOs and the commented out code is all over the place and there doesn't seem to be a clear structure for commented out code and sometimes there isn't even a comment regarding what the TODO actually refers to